DEVELOPMENT.md

```diff
@@ -193,6 +193,13 @@ The Console is the official management application of the stack. It can be used
 The OAuth app provides the necessary frontend for the OAuth provider of the stack. It is used e.g. to display the authorization screen that users get prompted with when they want to authorize a third-party app to access the stack.
 
 ### Building the frontend
+
+#### Prerequisites
+In order to build the frontend, you'll need the following:
+* Node JS, version 10.x (see [Development Environment](#development-environment))
```

Comment:
@furtiman What version causes issues? As far as I remember you had `10.12.*` ?

Reply:
@bafonins The one I had was 12.3.*

https://nodejs.org/en/ - They say that indeed the LTS is 10.x

Reply:
Ok, so 10 should be good then. @bafonins, maybe we should downgrade then rather, might introduce compatibility issues otherwise.